Lib/concurrent/futures/thread.py

```diff
@@ -158,7 +161,7 @@ def __init__(self, max_workers=None, thread_name_prefix='',
         self._initializer = initializer
         self._initargs = initargs
 
-    def submit(self, fn, /, *args, **kwargs):
+    def submit(self, fn, name='', /, *args, **kwargs):
```

Comment:
I think `None` would be a more common missing value here.

Reply:
yes, I will change it